scripts/prepare_sector_network.py

```diff
@@ -516,7 +516,7 @@ def add_carrier_buses(n, carrier, nodes=None):
         capital_cost=capital_cost,
     )
 
-    fossils = ["coal", "gas", "oil", "lignite"]
+    fossils = ["coal", "gas", "oil", "lignite", "uranium"]
     if options["fossil_fuels"] and carrier in fossils:
```

Comment:
```suggestion
    fossils = ["coal", "gas", "oil", "lignite"]
    if (options["fossil_fuels"] and carrier in fossils) or (carrier == "uranium"):
```
As `uranium` is not a fossil fuel, this would be less misleading.

Reply:
@daniel-rdt